src/core/stream_set.h

```diff
@@ -32,6 +32,11 @@ typedef struct QUIC_STREAM_TYPE_INFO {
     //
     uint16_t CurrentStreamCount;
 
+    //
+    // If th info is held in common by both endpoints (TP exchange is done)
+    //
+    BOOLEAN IsMutual;
```

Comment:
I don't understand this. I don't think it's necessary.

Reply:
I think you just want to check if the peer TP has been received then. No need to add an extra flag for that. Please see `PeerTransportParameterValid` on the connection.